docs/config/lua/config/font_rules.md

```diff
@@ -45,46 +48,79 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example combining Fira Code and Victor Mono;
 
 ```lua
 local wezterm = require 'wezterm'
-
--- A helper function for my fallback fonts
-function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
-  return wezterm.font_with_fallback(names, params)
-end
-
 return {
-  font = font_with_fallback 'Operator Mono SSm Lig Medium',
+  font = wezterm.font_with_fallback {
+    { family = 'FiraCode', weight = 'Light' },
+    -- Fallback that looks nice in small font sizes
+    'Terminus',
```

Comment:
in a similar spirit to my comment about removing `harfbuzz_features` from this example, let's also remove `Terminus` from here, as I think it steals focus from demonstrating the `font_rules`; `Terminus` being listed here doesn't affect the the `FiraCode` as the base font vs. `VictorMono` for italics that is the main benefit of spelling out these `font_rules`.

Reply:
As it currently is, it is a workable example, and that removing the stylistic sets and giving a clue that it works in small font sizes, removing Terminus that shows a different way of defining or adding fonts with wezterm.font_with_fallback, and having removed font_size from this in the past as well.

The example could very well define a few random fonts like Times New Roman and have the bare minimum without any tips and tricks, especially not a lua function to shorten the code with style and way of doing the structure that is incomplete for dealing with it as a string instead of a table.

Removed the harfbuzz_features, an entirely synthetic example mixing JetbrainsMono and VictorMono is my proposal.

Reply:
I appreciate the effort you've put into this so far, but it has reached the point where it is taking a lot more time and effort to work through than I'd like to spend on what should be a simple doc fix.

I'm going to make a pass over this page myself to clarify it.

Reply:
> I'm going to make a pass over this page myself to clarify it.

In the meantime this is an improvement for new users, which I still am.